generate_rss.py

```diff
@@ -13,55 +15,79 @@ def load_seen_comics():
             return json.load(file)
     return []  # If the file doesn't exist, return an empty list
 
-# Save seen comics to a file
+# Save seen comics to a file, limiting the size of the list
```

Comment:
The purpose of this .json file is to ensure that the RSS feed does not display the same xkcd comic until every comic has been shown at least once. I intend to implement code that will reset the .json file once the number of viewed comics matches the total number of available comics.

Reply:
Makes more sense, thanks.